venues/ICLR.cc/2019/Conference/webfield/bidWebfield.js

```diff
@@ -48,15 +48,28 @@ function load() {
     });
   });
 
-  var metadataNotesP = Webfield.getAll('/notes', {invitation: METADATA_INVITATION_ID}).then(function(allNotes) {
-    var metadataMap = {};
-    for (var i = 0; i < allNotes.length; i++) {
-      metadataMap[allNotes[i].forum] = allNotes[i].content.groups;
+  var userScoresP = Webfield.getAll('/notes', {invitation: USER_SCORES_INVITATION_ID}).then(function(scoreNotes) {
```

Comment:
Again we are getting all the notes for all the users. We need to get a single note, one option is to filter by content.user == profile.id and I can add an index in mongo. 

```
GET /notes?invitation=[invitaiton_id]&content.user=~profile_id
```

but content.user should an string.

Reply:
I'm not sure if this helps, but the readership permissions are set so that each reviewer or AC should only receive back one note from this query. does that make it more efficient? I'm not sure